include/jsonbuilder/JsonBuilder.h

```diff
@@ -1479,6 +1545,42 @@ Exchanges the contents of two JsonBuilder objects.
 */
 void swap(JsonBuilder&, JsonBuilder&) throw();
 
+/*
+UUID object, network byte order (big-endian), compatible with uuid_t.
+Does NOT use the same byte order as the Windows GUID type.
+*/
+struct UuidStruct
+{
+    char unsigned Data[16]; // Compatible with uuid_t from libuuid.
+};
+
+/*
+DateTime, number of 100ns intervals since 1601, compatible with Win32 FILETIME.
+*/
+struct TimeStruct
+{
+private:
+    using JSON_UINT32 = JsonInternal::JSON_UINT32;
+    using JSON_UINT64 = JsonInternal::JSON_UINT64;
+
+public:
+
+    JSON_UINT32 Low;
+    JSON_UINT32 High;
```

Comment:
Note: JsonBuilder only guarantees 32-bit alignment of values, so don't use a 64-bit type here. (This ends up being bitwise-compatible with FILETIME.)